Data/DataManagement.cs

```diff
@@ -0,0 +1,112 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.IO;
+using System.Linq;
+using System.Collections.Generic;
+using System.Configuration;
+using nightOwl.Properties;
+using nightOwl.Components;
+
+namespace nightOwl.Data
+{
+    public class DataManagement : IDataManagement
+    {
+        public List<Person> PersonsCatalog = new List<Person>();
+        public List<User> UsersCatalog = new List<User>();
+        public int UserID = 0;
+
+        private string DirectoryPath { get; set; } = Directory.GetParent(Directory.GetCurrentDirectory()).Parent.FullName + Settings.Default.DataFolderPath;
+        private string PersonsPath = Settings.Default.PersonsFileName;
+        private string UsersPath = Settings.Default.UsersFileName;
+
+
+        protected static DataManagement _obj;
+
+        private DataManagement()
+        {
+            if (!Directory.Exists(DirectoryPath))
+                Directory.CreateDirectory(DirectoryPath);
+
+            if (!File.Exists(DirectoryPath + PersonsPath))
+            {
+                var newFile = File.Create(DirectoryPath + PersonsPath);
+                newFile.Close();
+
+                File.WriteAllText(DirectoryPath + PersonsPath, "[]");
+            }
+            if (!File.Exists(DirectoryPath + UsersPath))
+            {
+                var newFile = File.Create(DirectoryPath + UsersPath);
+                newFile.Close();
+
+                File.WriteAllText(DirectoryPath + UsersPath, "[]");
+            }
+        }
+
+        public static DataManagement GetInstance()
+        {
+            if (_obj == null)
+                _obj = new DataManagement();
+
+            return _obj;
+        }
+
+        public List<Person> GetPersonsCatalog()
+        {
+            return PersonsCatalog;
```

Comment:
You should not return the whole list which you can edit. I guess you are making a singleton repository, so singleton part is kind of okay (read the comment on Lazy above), but repository part is not really. You should use something like `IReadonlyList<Person>` as return type

Reply:
Unless you intend other code to change your list without knowing it in `DataManagement`